client/dashboard/index.js

```diff
@@ -14,17 +14,26 @@ import Header from 'header';
 import Leaderboards from './leaderboards';
 import { ReportFilters } from '@woocommerce/components';
 import StorePerformance from './store-performance';
+import TaskList from './task-list';
 
 export default class Dashboard extends Component {
 	render() {
 		const { query, path } = this.props;
+		// @todo This should be replaced by a check of tasks from the REST API response from #1897.
+		const requiredTasksComplete = false;
```

Comment:
Should we go ahead and set this to `true` for now, since others will be working on the analytics side of the dashboard? It'll add an extra step for us until #1897 is addressed, but otherwise we'll be adding an extra step for the rest of the team.

Reply:
Sure thing.  Set to `true` to hide by default until #1897 is taken care of.